docs/sinco/v3.x/2_tutorials/2_page/6_custom_assertions.md

```diff
@@ -0,0 +1,63 @@
+# Custom Assertions
+
+## Table of Contents
+
+- [Before You Get Started](#before-you-get-started)
+- [Folder Structure End State](#folder-structure-end-state)
+- [Steps](#steps)
+- [Verification](#verification)
+
+## Before You Get Started
+
+`Page` provides some utility methods that act as wrappers for Deno's std/testing
+assertions. These methods will run a query on the DOM and make an assertion. You
+do not need to assert these yourself.
+
+The methods it provides are:
+
+- `assertSee` - This method will assert that the given text exists inside the
+  page
```

Comment:
```suggestion
The method it provides is:

- `assertSee` - This method will assert that the given text exists inside the
  page.
```

Reply:
kinda against this, cause the wording allows for expansion should we ever add more, we dont need to update the text

Reply:
i figured that's where you were going with this, but suggested anyways. alright ! LGTM